web/themes/custom/itkdev/itkdev_project_theme/templates/node--project.html.twig

```diff
@@ -16,14 +16,16 @@
 {% extends '@itkdev_project_theme/content-wrapper.html.twig' %}
 
 {% block title %}
-  <div class="inline-flex items-center"><h1 class="dark:text-white inline-block">{{ label }}</h1><a href="{{ url('entity.node.edit_form', {node: node.id}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a></div>
+  <div class="inline-flex items-center">
+    <h1 class="dark:text-white inline-block">{{ label }}</h1>
+    <a href="{{ url('entity.node.edit_form', {node: node.id}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a>
```

Comment:
I think we're missing an access check here. Everybody can see a project, but only group members can edit:

```suggestion
    {% if node.access('update') %}
    <a href="{{ url('entity.node.edit_form', {node: node.id}) }}" class="inline-block mx-3 btn-icon bg-black text-white hover:bg-stone-700"><i class="fa-solid fa-pencil"></i></a>
    {% endif %}
```

Reply:
But yeah, somethign needs to be done

Reply:
Hmm …